### ISSUE TITLE
Acknowledgements section always gets line numbers in v6.3.1 [Which breaks arXiv submissions]

### ISSUE THREAD

**Issue by zpleunis** [AASJournals/AASTeX60#130]
Using aastex631.cls the acknowledgements section always get line numbers (i.e., also without `linenumbers` selected as an option). 

I was able to fix this by replacing
`\begin{internallinenumbers}` and `\end{internallinenumbers}` with `\ifnumlines\begin{internallinenumbers}\fi` and `\ifnumlines\end{internallinenumbers}\fi` on lines 1240, 1249, 1296 and 1305 in the .cls file.

**Comment by gregschwarz** [comment]
I have not been able to confirm this locally. Can you let me know what style options, if any, you are using?

**Comment by zpleunis** [comment]
Yes, I use `\documentclass[twocolumn]{aastex631}`. The only package that I import outside of the class file is `amsmath`.

**Comment by seawander** [comment]
I did observe what @zpleunis saw ~2 months ago, and recorded it in my own mod of the aster package https://github.com/seawander/aastex_pwned/commit/7854f159290a7b3672e2ddec16075c7d881a5622 . Thanks for the fix, @zpleunis !

**Comment by mknote88** [comment]
I sent an email regarding this issue on March 5 and got a response (from Greg Schwarz, actually, who I assume is the same as the above).  He also found that it was only replicated when `amsmath` was imported, which was true in my case as well.  It _does_ seem to be a conflict relating to `amsmath`.

**Comment by GijsMulders** [comment]
I also experienced this issue today (and using amsmath)
astro-ph actually rejects your manuscript if it includes line numbers anywhere, so might be worth fixing.

The solution in the opening posts works for me, replacing those for lines.

**Comment by gregschwarz** [comment]
This is in our list for the next version. Note that you also do not have to use the \begin{acknowledgment} ... \end{acknowledgment} environment. It is primarily needed to blank out the acknowledgments when using the "anonymous" style option for dual anonymous review (DAR).

**Comment by gregschwarz** [comment]
Yes, or just create a new \section{Acknowledgment} if you really need a section header for some reason.

**Comment by jpinedaf** [comment]
Just to check, is this bug been addressed? As mentioned earlier, arXiv rejects these submissions.

**Comment by parkus** [comment]
I also received an arXiv rejection due to this, so glad it is on the list to fix.

My workaround was to simply remove the acknowledgments environment and instead just separate that paragraph from the end of the main text body with a few `\\` line breaks.

**Comment by sedonaprice** [comment]
My fix: wrap both lines 1239-1241 and lines 1248-1250 with `\ifnumlines` .... `\fi`, then the acknowledgements follows the document-level line numbering setting.

**Comment by matiscke** [comment]
ArXiv rejected our paper due to this bug. I'm glad this issue is on the list to be fixed; I suspect it affects many users. Any news on when we can expect to see a new aastex release? I appreciate the hard work that goes into this.

**Comment by augustfly** [comment]
We are in fact beginning to scope out the next release, but dev hasn't started and given the issues, this might be big one. I'm really sorry this bit your arXiv release @matiscke! There are so many nests of dependencies that some clean out is required.
